Update nightly smoke test

**Bug report**
The nightly smoke tests have been failing for the ubuntu builds (MacOS passes)

The error message: 
```
_______________ ERROR collecting tests/layup/test_cpp_import.py ________________
ImportError while importing test module '/home/runner/work/layup/layup/tests/layup/test_cpp_import.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.12.9/x64/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/layup/test_cpp_import.py:1: in <module>
    from layup import routines
src/layup/routines/__init__.py:3: in <module>
    from _layup_cpp._core import __doc__, __version__, hello_world
E   ImportError: librebound.so: cannot open shared object file: No such file or directory
```

**Before submitting**
Please check the following:

- [ ] I have described the situation in which the bug arose, including what code was executed, information about my environment, and any applicable data others will need to reproduce the problem.
- [ ] I have included available evidence of the unexpected behavior (including error messages, screenshots, and/or plots) as well as a description of what I expected instead.
- [ ] If I have a solution in mind, I have provided an explanation and/or pseudocode and/or task list.

--- a/.github/workflows/smoke-test.yml
+++ b/.github/workflows/smoke-test.yml
@@ -33,7 +33,6 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         git submodule update --init
-        export LD_LIBRARY_PATH=$(pwd):$LD_LIBRARY_PATH
         pip install -e .[dev]
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: List dependencies
@@ -41,4 +40,5 @@ jobs:
         pip list
     - name: Run unit tests with pytest
       run: |
+        export LD_LIBRARY_PATH=$(pwd):$LD_LIBRARY_PATH
         python -m pytest

